### DISCUSSION TITLE
Logseq support?

### DISCUSSION THREAD

**Discussion by dam5h** [Feel-ix-343/markdown-oxide#74] [Q&A] [answered]
Curious what the state of logseq support is and if there are any how to guides?  I use helix as an editor.  Thanks!

**Comment by Feel-ix-343** [comment]
Hey!

Logseq syntax is really only somewhat supported. All logseq links should work, but note that moxide delivers link completions for more than just files; you can change this in the config specified in the readme. Also logsec block links `((identifier))` are not supported. You can block link using `[[ fuzzy search` which returns obsidian block link syntax. However I don't know how well this works on helix because helix does not fully implement the lsp; it works well in vscode and neovim right now.

**Reply by dam5h** [reply, marked as answer]
Cool, thanks!

I am hoping the missing part of the lsp lands in helix soon, seems as though the maintainer just picked it up, or at least I saw some chatter about it recently.

I see you are the one that put the ticket in actually.

https://github.com/helix-editor/helix/issues/9797

**Reply by dam5h** [reply]
@Feel-ix-343 

To follow up on this I am trying to use the wikilinks style in a daily journal.

I can get [[today]] to turn into [[2024-04-30]] by trying hte completion a second time (your mentioned workaround in the helix issue), but after it completes and I leave it as such, it just says "Unrecognized reference".

Is this a config issue?

![image](https://github.com/Feel-ix-343/markdown-oxide/assets/6738286/9b412771-18fb-48ea-9ee6-60aaceda4629)

Cheers!

**Reply by Feel-ix-343** [reply]
Hey! I'm happy to hear you found a workaround.

I'm wondering if you have created the daily note file? If not you should be able to through a code action. 

Let me know how this works.

**Reply by dam5h** [reply]
I wanted to have all my daily stuff in one big long file.  I can create separate date files via code action but I was hoping this could just convert into a header instead of being a link.

For now I am just removing the brackets after it populates.

**Reply by Feel-ix-343** [reply]
Ok interesting.

Do you think some non-link-completion date completions would be helpful? We could implement slash commands so you would get a date completion of 2024-05-03 for /today for example

**Reply by dam5h** [reply]
Yup, that would be great!

One argument for the 1 big file is you can easily yank lines for tasks not completed and move them around, not having to jump around to different files can be nice for simple workflows.

**Reply by dam5h** [reply]
@Feel-ix-343, I am happy to investigate how difficult it would be to submit a PR for this as well, I will check if we have any existing slash commands now.

Thanks!

**Reply by Feel-ix-343** [reply]
Hey thank you!

This would be very helpful. I cannot dedicate much time to development right now. Note that the code is not super great right now, and I am working on a large scoped refactor. This means both that contributing may be difficult but also that hacky ways of doing things are okay.

It would be super helpful if you open a pr; its okay if you haven't written any code yet
